loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -181,7 +186,19 @@ public LoadFlowParameters setSpecificCompatibility(boolean specificCompatibility
     }
 
     public LoadFlowParameters copy() {
-        return new LoadFlowParameters(this);
+        byte[] bytes = writeInMemory();
+        try (ByteArrayInputStream bais = new ByteArrayInputStream(bytes)) {
+            return JsonLoadFlowParameters.read(bais);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    private byte[] writeInMemory() {
+        try (ByteArrayBuilder byteArrayBuilder = new ByteArrayBuilder()) {
```

Comment:
Why do you use `ByteArrayBuilder`?  Why don't you use `ByteArrayOutputStream`?

Reply:
From doc, it seems like it is a better usage here.
> Since version 1.5, also implements OutputStream to allow efficient aggregation of output content as a byte array, similar to how ByteArrayOutputStream works, but somewhat more efficiently for many use cases.

Reply:
We could probably keep `ByteArrayBuilder` then...